perllib/FixMyStreet/DB/Result/Body.pm

```diff
@@ -204,7 +204,7 @@ sub get_cobrand_handler {
 }
 
 sub calculate_average {
-    my ($self) = @_;
+    my ($self, $threshold) = @_;
```

Comment:
Perhaps here have a safety `$threshold ||= 0;` line, and then you don't need to make sure it's 0 when calling the function in the two places.